### PR TITLE
Fix: lowercase legacy numbers

### PR DESCRIPTION
**Pull request by akaJSaunders**
### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are commited to keeping commit history clean, consistent and linear. To achive this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/PenaltyDetailsController.ts [hunk 1]
```diff
@@ -24,12 +24,12 @@ const navigation = {
 };
 
 const sanitizeForm = (body: PenaltyIdentifier) => {
-    const legacyPrefixChars: string = 'PEN';
-    const penalty = body.penaltyReference;
+    const legacyPrefix: string = 'PEN';
+    const penalty = body.penaltyReference.toUpperCase();
```
Comment (n-r-b) [inline src/controllers/PenaltyDetailsController.ts:28]: i've noticed we are setting the penalty number to uppercase in sanitizeLegacyPenalty(), do we still need to do this?

Comment (n-r-b) [reply src/controllers/PenaltyDetailsController.ts:28]: Can you also add the type please

File: src/controllers/PenaltyDetailsController.ts [hunk 2]
```diff
@@ -24,12 +24,12 @@ const navigation = {
...
 
     return {
         companyNumber: sanitizeCompany(body.companyNumber),
-        penaltyReference: penalty.startsWith(legacyPrefixChars) ? sanitizeLegacyPenalty(penalty) : penalty.toUpperCase()
```
Comment (franciscog-kainos) [inline src/controllers/PenaltyDetailsController.ts:32]: the `sanitizeLegacyPenalty` already upper cases the reference. You could have left this as it was or am I missing something?

Comment (n-r-b) [reply src/controllers/PenaltyDetailsController.ts:32]: i think he's doing this because `startsWith()` is case sensitive

**Pull request merged**